addons/l10n_zm_account/data/template/account.account-zm.csv

```diff
@@ -0,0 +1,90 @@
+"zm_account_4560000","License costs","456","expense","False"
+"zm_account_4600000","Telephone & Fax","46","expense","False"
+"zm_account_4650000","Travel & Accommodation","465","expense","False"
+"zm_account_4700000","Loss on Sale of Non Current Assets","47","expense","False"
+"zm_account_5200000","Retained Income / (Accumulated Loss)","52","liability_current","False"
+"zm_account_5500000","Long Term Liabilities","55","liability_non_current","False"
+"zm_account_5600000","Instalment Sale Creditors Other","56","liability_non_current","False"
+"zm_account_6100000","Land & Buildings - Net Value","610000","asset_fixed","False"
+"zm_account_6100010","Land & Buildings - At Cost","610001","asset_fixed","False"
+"zm_account_6100020","Land & Buildings - Accumulated Depreciation","610002","asset_fixed","False"
+"zm_account_6200000","Motor Vehicles - Net Value","620000","asset_fixed","False"
+"zm_account_6200010","Motor Vehicles - At Cost","620001","asset_fixed","False"
+"zm_account_6200020","Motor Vehicles - Accumulated Depreciation","620002","asset_fixed","False"
+"zm_account_6250000","Computer Equipment - Net Value","625000","asset_fixed","False"
+"zm_account_6250010","Computer Equipment - At Cost","625001","asset_fixed","False"
+"zm_account_6250020","Computer Equipment - Accumulated Depreciation","625002","asset_fixed","False"
+"zm_account_6300000","Office Equipment - Net Value","630000","asset_fixed","False"
+"zm_account_6300010","Office Equipment - At Cost","630001","asset_fixed","False"
+"zm_account_6300020","Office Equipment - Accumulated Depreciation","630002","asset_fixed","False"
+"zm_account_6350000","Furniture & Fittings - Net value","635000","asset_fixed","False"
+"zm_account_6350010","Furniture & Fittings - At Cost","635001","asset_fixed","False"
+"zm_account_6350020","Furniture & Fittings - Accumulated Depreciation","635002","asset_fixed","False"
+"zm_account_6600000","Other Fixed Assets - Net Value","660000","asset_fixed","False"
+"zm_account_6600010","Other Fixed Assets - At Cost","660001","asset_fixed","False"
+"zm_account_6600020","Other Fixed Assets - Accumulated Depreciation","660002","asset_fixed","False"
+"zm_account_7000000","Goodwill / Other Intangible Assets","7","asset_fixed","False"
+"zm_account_7100000","Investments","71","asset_current","False"
+"zm_account_7700000","Inventory Control Account","77","liability_current","False"
+"zm_account_8000000","Customers","8","asset_receivable","True"
+"zm_account_8100000","Customes (POS)","81","asset_receivable","True"
+"zm_account_8200000","Sundry Customers","82","asset_current","False"
+"zm_account_8300000","VAT Purchase Tax Control Account","83","asset_current","False"
+"zm_account_8300001","Withholding Tax on Sales","8300001","asset_current","False"
+"zm_account_8300002","Withholding Tax on Sales - Transition Account","8300002","asset_current","False"
+"zm_account_8310000","VAT Receivable - Closing Entry","831","asset_receivable","True"
```

Comment:
@Laura-labe Should this one be receivable?

Reply:
@jco-odoo It depends on when the VAT credit is expected to be reimbursed by the authorities. But it's true that receivable category is more adequate for customers credit rather than authorities credit. Let's change the "VAT Receivable - Closing Entry" category to Current Assets one.